README.md

```diff
@@ -113,21 +113,36 @@ Asynchronous task are handled with [coroutines](https://developer.android.com/ko
         /src
             /main
                 /java
-                    /ar.scacchipa.twittercloneapp
-                        /di
-                        /component
-                        /domain
-                        /ui
-                        /repository
-                        /datasource
+                    / data
+                        / model
+                        / sourcedata
+                        / repository
+                    / domain
+                        / model
+                        / usecase
+                    / presentation
+                       / start
```

Comment:
please update de readme.

Reply:
The README was updated. Thes folder structure description was fixed.